src/main/java/bowling/domain/Player.java

```diff
@@ -0,0 +1,25 @@
+package bowling.domain;
+
+import bowling.domain.exception.IllegalPlayerName;
+
+public class Player {
+    public static int NAME_MAX_SIZE = 3;
```

Comment:
```suggestion
    public static final int NAME_MAX_SIZE = 3;
```
상수이니 final 키워드를 사용해주세요!

Reply:
ResultView, Round, Start, Strike 의 상수에도 같은 코멘트 드리겠습니다. 🙂

Reply:
Player NAME_MAX_SIZE는 final 키워드를 빠드리신거 같아요. 🙂